src/impl_core.cpp

```diff
@@ -233,6 +247,15 @@ auto construct_daxa_physical_device_properties(VkPhysicalDevice physical_device)
         ret.mesh_shader_properties.value.prefers_compact_vertex_output = static_cast<daxa_Bool8>(vk_physical_device_mesh_shader_properties_ext.prefersCompactVertexOutput);
         ret.mesh_shader_properties.value.prefers_compact_primitive_output = static_cast<daxa_Bool8>(vk_physical_device_mesh_shader_properties_ext.prefersCompactPrimitiveOutput);
     }
+    if (host_image_copy_supported)
+    {
+        ret.host_image_copy_properties.has_value = 1;
+        std::memcpy(
```

Comment:
As mentioned in file device.h, we should prob skip the image layouts as well in this memcpy. Luckily they are at the start of the struct so we can keep this code by adding a larger offset and smaller size